src/sys/windows.rs

```diff
@@ -432,6 +432,16 @@ impl Socket {
         unsafe { self.setsockopt(IPPROTO_IP, IP_TTL, ttl as c_int) }
     }
 
+    #[cfg(feature = "all")]
+    pub fn set_bound_interface(&self, _if_index: c_uint) -> io::Result<()> {
```

Comment:
These two functions should just be removed. Behind the `all` feature flag is OK to have missing functions.